### PR TITLE
Threat Intelligence project 4 - beta

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new beta article:
https://help.sumologic.com/docs/platform-services/threat-intelligence-indicators

At GA, parts of this article will be moved into other articles as indicated by comments.

Note: This article will eventually go under a "Threat Intelligence" section under "[Platform Services](https://help.sumologic.com/docs/platform-services/)". But for now, for this closed beta, we're placing it directly under "Platform Services". 

Issue number: Asana ticket [Document Threat Intelligence tab on the Logs page](https://app.asana.com/0/1183741177809740/1205156453843959/f)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/threat-intelligence-indicators.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Threat intelligence indicators can help a security analysts leverage a large body of information to surface potential threats. For example, say that a threat intelligence database has an indicator that correlates a certain IP address with known malicious activity. Because of this correlation, analysts can assume log messages with that IP address are more likely to be part of a real cyber attack.
+
+You can add threat intelligence indicators from a number of sources, including CrowdStrike, TAXII, ThreatQ, iDefense, and many others. And threat intelligence indicators imported to Sumo Logic not only integrate with your existing core Sumo Logic deployment, but also Cloud SIEM and Cloud SOAR. (For information about how to add additional threat intelligence indicators for Cloud SIEM, see [Create a Custom Threat Intelligence Source](/docs/cse/administration/create-custom-threat-intel-source).)
+
+## Threat Intelligence tab
+
+Use the **Threat Intelligence** tab to add and manage threat intelligence indicators. To access the tab, in Sumo Logic go to **Manage Data > Logs > Threat Intelligence**.
```
Comment (kimsauce) [inline docs/manage/threat-intelligence-indicators.md:18]:
```suggestion
Use the **Threat Intelligence** tab to add and manage threat intelligence indicators. To access the tab, go to **Manage Data > Logs > Threat Intelligence**.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
beautifully done! just one minor suggestion.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Review by kimsauce — APPROVED** [review]

File: docs/manage/threat-intelligence-indicators.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+1. **Indicators**. The number of threat intelligence indicators included in the file. 
+
+## Add threat intelligence indicators
+
+To add threat intelligence indicators, you must upload files containing the indicators in a format that can be consumed by Sumo Logic.
+
+1. In Sumo Logic, go to **Manage Data > Logs > Threat Intelligence**.
+1. Click **Add Indicators**. The dialog displays. <br/><img src={useBaseUrl('img/manage/threat-intelligence-add-indicators.png')} alt="Add threat intelligence indicators" style={{border: '1px solid black'}} width="500" />
+1. Select the format of the file to be uploaded:
+    * **Sumo normalized JSON**. A JSON file [normalized](/docs/cse/schema/parsing-language-reference-guide#normalizing) for use in Sumo Logic. Your file must include the following attributes:
```
Comment (knamburi-sumo) [inline docs/manage/threat-intelligence-indicators.md:36]: This is more accurate info: 
https://api.sumologic.com/docs/#operation/uploadNormalizedIndicators

Stix:
https://api.sumologic.com/docs/#operation/uploadStixIndicators
We have a meeting on how to handle stix indicator uploads. From UI perspective only the fields in indicators object which are marked required are required with file upload.

Comment (jpipkin1) [reply docs/manage/threat-intelligence-indicators.md]: @knamburi-sumo, thank you very much for that information! I added links to the APIs with the [Add API links](https://github.com/SumoLogic/sumologic-documentation/pull/2936/commits/010d57b05009ab378f517e67221a77f4cffa016b) commit.

**Review by kimsauce — APPROVED** [review]

File: cid-redirects.json [hunk 3]
```diff
@@ -1441,6 +1441,7 @@
   "/cid/1": "/docs/search/get-started-with-search/build-search/search-syntax-overview",
   "/cid/0100": "/docs/manage/security/installation-tokens",
   "/cid/0020": "/docs/manage/health-events",
+  "/cid/0020001": "/docs/manage/threat-intelligence-indicators",
```
Comment (jpipkin1) [inline cid-redirects.json:1444]:
```suggestion
  "/cid/0020001": "/docs/platform-services/threat-intelligence-indicators",
```

**Comment by jpipkin1** [comment]
Closed this PR because we're replacing it with PR #3368.

**Pull request closed without merging**